Python/dynload_win.c

```diff
@@ -18,21 +18,25 @@
 #define PYD_DEBUG_SUFFIX ""
 #endif
 
-#ifdef PYD_PLATFORM_TAG
-#define PYD_TAGGED_SUFFIX PYD_DEBUG_SUFFIX ".cp" Py_STRINGIFY(PY_MAJOR_VERSION) Py_STRINGIFY(PY_MINOR_VERSION) "-" PYD_PLATFORM_TAG ".pyd"
+#ifndef __MINGW32__
+    #ifdef PYD_PLATFORM_TAG
+    #define PYD_TAGGED_SUFFIX PYD_DEBUG_SUFFIX ".cp" Py_STRINGIFY(PY_MAJOR_VERSION) Py_STRINGIFY(PY_MINOR_VERSION) "-" PYD_PLATFORM_TAG ".pyd"
```

Comment:
given the pyd change this would all be a bit cleaner if we would set `PYD_PLATFORM_TAG` before.. but not sure how easy that is.

Reply:
> but not sure how easy that is.

I think that should be simple enough.